src/main.js

```diff
@@ -756,10 +761,11 @@ function switchMenuIconImages(icons, isDarkMode) {
   }
 }
 
-function setDeeplinkingUrl(url) {
-  if (scheme) {
-    deeplinkingUrl = url.replace(new RegExp('^' + scheme), 'https');
+function getDeeplinkingUrl(url) {
+  if (!scheme || !Utils.isValidURL(url)) {
```

Comment:
Can we invert this condition? It's a bit unclear. Also it would be nice that `null` is the default.